pyproject.toml

```diff
@@ -0,0 +1,109 @@
+[build-system]
+requires = ["scikit-build-core"]
+build-backend = "scikit_build_core.build"
+
+[project]
+name = "itk-fastbilateral"
+version = "0.1.0"
+description = "Faster implementation of the itkBilateralImageFilter. From the insight journal submission: https://www.insight-journal.org/browse/publication/692."
```

Comment:
The `.` at the end of the sentence breaks the link.

Reply:
PR #6.